separator.go

```diff
@@ -16,284 +16,28 @@
 
 package main
 
-import (
-	"strings"
-)
+import "github.com/apstndb/gsqlsep"
 
 type delimiter int
 
 const (
-	delimiterUndefined delimiter = iota
-	delimiterHorizontal
-	delimiterVertical
+	delimiterUndefined  = ""
+	delimiterHorizontal = ";"
+	delimiterVertical   = `\G`
 )
 
 type inputStatement struct {
 	statement string
-	delim     delimiter
-}
-
-func (d delimiter) String() string {
-	switch d {
-	case delimiterUndefined:
-		return ""
-	case delimiterHorizontal:
-		return ";"
-	case delimiterVertical:
-		return `\G`
-	}
-	return ""
+	delim     string
 }
 
 func separateInput(input string) []inputStatement {
```

Comment:
If we keep this function, why not calling this function in other places? (not directly call gsqlsep functions)